packages/apollos-church-api/src/data/content-items/__tests__/resolvers.tests.js

```diff
@@ -0,0 +1,230 @@
+import { graphql } from 'graphql';
+import { fetch } from 'apollo-server-env';
+
+import { createGlobalId } from '@apollosproject/server-core';
+import { createTestHelpers } from '@apollosproject/server-core/lib/testUtils';
+import ApollosConfig from '@apollosproject/config';
+
+import {
+  mediaSchema,
+  themeSchema,
+  scriptureSchema,
+} from '@apollosproject/data-schema';
+
+// we import the root-level schema and resolver so we test the entire integration:
+import {
+  ContentChannel,
+  Sharable,
+  RockConstants,
+} from '@apollosproject/data-connector-rock';
+
+import * as ContentItem from '../index';
+
+class Scripture {
+  // eslint-disable-next-line class-methods-use-this
+  initialize() {}
+
+  // eslint-disable-next-line class-methods-use-this
+  getScriptures() {
+    return [
+      {
+        html: `<p class="p"><span data-number="1" class="v">1</span>The Song of songs, which is Solomon’s.</p>`,
+      },
+    ];
+  }
+}
+
+const { getSchema, getContext } = createTestHelpers({
+  ContentChannel,
+  ContentItem,
+  Sharable,
+  UniversalContentItem: {
+    dataSource: ContentItem.dataSource,
+  }, // alias
+  DevotionalContentItem: {
+    dataSource: ContentItem.dataSource,
+  }, // alias
+  Scripture: {
+    dataSource: Scripture,
+  },
+  RockConstants,
+});
+// we import the root-level schema and resolver so we test the entire integration:
+
+ApollosConfig.loadJs({
+  ROCK: {
+    API_URL: 'https://apollosrock.newspring.cc/api',
+    API_TOKEN: 'some-rock-token',
+    IMAGE_URL: 'https://apollosrock.newspring.cc/GetImage.ashx',
```

Comment:
Our Rock Instance. Also, can this information just come from the config that already has this information in it?

Reply:
these info doesn't matter in the test so probably added work without any benefit.